F1WM/DatabaseModel/F1WM/News.cs

```diff
@@ -7,11 +7,11 @@ public class News
 	{
 		public uint Id { get; set; }
 		public DateTime Date { get; set; }
-		public byte Type { get; set; }
+		public byte TypeId { get; set; }
 		public int? NewsModified { get; set; }
 		public string PosterName { get; set; }
-		public uint TopicId { get; set; }
-		public NewsTopic Topic { get; set; }
+		public uint TagId { get; set; }
+		public NewsTag Tag { get; set; }
```

Comment:
Here the distinction between the main tag and other tags is even more necessary. Please rename Tag to MainTag and TagId to MainTagId.